lib/splitio-api-binding.js

```diff
@@ -53,11 +54,38 @@ class SplitioApiBinding {
     }
   }
 
+  /**
+   * Polls the Split.io API until since and till timestapms are the same.
+   */
+  async getAllChanges ({ path }) {
+    let since = SINCE_VALUE_FOR_FIRST_REQUEST
+    let results = await this.httpGet({ path, since })
+    const allChanges = [results]
+    // The till value represents the timestamp of the last change included in the response.
+    let till = results.till
+    while (since !== till) {
```

Comment:
Are these values guaranteed to converge? I am worried that there would be a case where this runs infinitely.

Reply:
i think they will -- it would only run a long time if there's a huge number splits

from the doc from split:
```If “since” value equals “till” value, there are no more changes.
Ideally you would keep fetching changes until this condition is fulfilled, instead of just assuming one request will get you everything. 
After this condition is true, you periodically poll for updates.
There will only be updates to the data if this condition is NOT true on a given response.
```

Reply:
The potentially infinite loop makes me nervous.  E.g., if split.io API has a bug, that bubbles up as a infinite loop in our product code.  Can we add a limit on the loop?